frontend/packages/db-structure/src/parser/supportedFormat/schema.ts

```diff
@@ -6,4 +6,10 @@ export const supportedFormatSchema = v.union([
   v.literal('prisma'),
 ])
```

Comment:
I thought it would be better to use an array of supportedFormats as SSoT and use picklist.

https://valibot.dev/api/picklist/

eg: `export const supportedFormatSchema = v.picklist(supportedFormats)`

Reply:
Got it! It looks like the argument for `v.picklist` needs to be assigned as an array of string literals. I’ve fixed it accordingly!